### PR TITLE
Add Event Tax when placing bounties with one

### PR DESCRIPTION
**Pull request by Marenthyu**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -5589,23 +5589,26 @@ def do_command(self, command, args, sender, channel, tags, isWhisper=False):
 
                             # if it passed all of those checks it should be good to go.
                             # penalize them for reducing a bounty above regular cap?
+                            amount_changed = 0
                             if points_delta < 0 and old_bounty > rarity_cap:
                                 change_above_cap = min(-points_delta, old_bounty - rarity_cap)
-                                addPoints(tags['user-id'], (change_above_cap // 2 + (-points_delta - change_above_cap))*effectiveMultiplier)
+                                amount_changed = (change_above_cap // 2 + (-points_delta - change_above_cap))*effectiveMultiplier
+                                addPoints(tags['user-id'], amount_changed)
```
Comment (Dabomstew) [inline main.py:5596]: if we're being pedantic, push this out of the if/else

File: main.py [hunk 2]
```diff
@@ -5589,23 +5589,26 @@ def do_command(self, command, args, sender, channel, tags, isWhisper=False):
...
                             else:
-                                addPoints(tags['user-id'], -points_delta*effectiveMultiplier)
+                                amount_changed = -points_delta*effectiveMultiplier
+                                addPoints(tags['user-id'], amount_changed)
```
Comment (Dabomstew) [inline main.py:5599]: which lets this be deduplicated

File: main.py [hunk 3]
```diff
@@ -5589,23 +5589,26 @@ def do_command(self, command, args, sender, channel, tags, isWhisper=False):
...
 
                             if myorderinfo is None:
                                 cur.execute(
                                     "INSERT INTO bounties (userid, waifuid, amount, status, created, eventMultiplier) VALUES(%s, %s, %s, 'open', %s, %s)",
                                     [tags['user-id'], waifu['id'], amount, current_milli_time(), eventMultiplier])
-                                self.message(channel, "%s, you placed a new bounty on [%d] %s for %d points." % (
-                                    tags['display-name'], waifu['id'], waifu['name'], amount), isWhisper)
+                                self.message(channel, "%s, you placed a new bounty on [%d] %s for %d points.%s" % (
+                                    tags['display-name'], waifu['id'], waifu['name'], amount, "" + ((" (costing you " + (-amount_changed) + " points due to having this event card already.)") if effectiveMultiplier != 1.0 else "")), isWhisper)
```
Comment (Dabomstew) [inline main.py:5606]: why do we need to append to a blank string here?

File: main.py [hunk 4]
```diff
@@ -5589,23 +5589,26 @@ def do_command(self, command, args, sender, channel, tags, isWhisper=False):
...
                             else:
                                 cur.execute("UPDATE bounties SET amount = %s, updated = %s WHERE id = %s",
                                             [amount, current_milli_time(), myorderinfo[0]])
-                                self.message(channel, "%s, you updated your bounty on [%d] %s to %d points." % (
-                                    tags['display-name'], waifu['id'], waifu['name'], amount), isWhisper)
+                                self.message(channel, "%s, you updated your bounty on [%d] %s to %d points.%s" % (
+                                    tags['display-name'], waifu['id'], waifu['name'], amount, "" + ((" (costing you " + (-amount_changed) + " points due to having this event card already.)") if effectiveMultiplier != 1.0 else "")), isWhisper)
```
Comment (Dabomstew) [inline main.py:5611]: see above

**Review by Dabomstew — COMMENTED** [review]
Function seems fine, here are some pedant code comments anyway.

**Review by world87 — APPROVED** [review]

**Pull request merged**